test/remote/gateways/remote_orbital_test.rb

```diff
@@ -829,6 +829,37 @@ def test_successful_verify
     assert_equal 'No reason to decline', response.message
   end
 
+  def test_successful_store
+    options = @options.merge(store: true)
```

Comment:
considering the previous comments, this test looks like a purchase with an option to store instead of a normal store. 

If you add use a store method instead of a purchase I think this test will change.